apps/desktop/src/components/v3/StackTab.svelte

```diff
@@ -14,69 +14,65 @@
 		selected: boolean;
 	};
 
+	const { projectId, tab, first, last, selected }: Props = $props();
+
 	let kebabMenuTrigger = $state<HTMLElement>();
-	let contextMenuEl = $state<ContextMenu>();
-	let isContextMenuOpen = $state(false);
+	let contextMenu = $state<ContextMenu>();
 
-	const { projectId, tab, first, last, selected }: Props = $props();
+	let nameEl = $state<HTMLDivElement>();
+	let nameWidth = $state<number>();
+
+	$effect(() => {
+		if (nameEl) {
+			nameWidth = nameEl.offsetWidth - 1;
+		}
+	});
```

Comment:
Hmm, I redid the tabs just yesterday and made it css only. What is the issue you are solving here?